cmd/it.go

```diff
@@ -1,58 +1,87 @@
 package cmd
 
 import (
+	"errors"
 	"fmt"
 	"os"
 	"strconv"
+	"strings"
+	"sync"
 
 	"github.com/cypherpunkarmory/punch/tunnel"
 	"github.com/cypherpunkarmory/punch/utilities"
 	"github.com/reiver/go-telnet"
 	"github.com/spf13/cobra"
 )
 
+type tunnelConf struct {
+	port        int
+	forwardType string
+}
+
 var itCmd = &cobra.Command{
 	Use:   "it [ports]",
 	Short: "Expose a web server on the ports you specify",
-	Long:  `To expose a web server on port 80 punch it 80`,
-	Args:  cobra.ExactArgs(1),
+	Long: `To expose a http web server on port 80 and https web server on port 443
```

Comment:
I think these are confusing instructions, since the ports your provide are _your local ports_ and are unlikely to be 80/443.   I would reword this as:

```
To expose an http web server on your computer listening on port 5000 and an https web server listening on port 5001 - punch it http:5000 https:5001
```